src/ui/wxWidgets/passwordsafeframe.cpp

```diff
@@ -2779,20 +2799,25 @@ void PasswordSafeFrame::OnOpenRecentDB(wxCommandEvent& evt)
   {
     case PWScore::SUCCESS:
       m_core.ResumeOnDBNotification();
+      CreateMenubar();  // Recreate the menu with updated list of most recently used DBs
```

Comment:
Is this the desired behavior that a new opened DB appears immediately in the list of most recently used DBs?